ui/src/styles/app.css

```diff
@@ -31,15 +31,17 @@
 }
 
 .site-layout {
-    padding: 0 50px;
+    /*background-color: #fff;*/
```

Comment:
Remove this pls if we do not need it any more.

Reply:
fixed.